examples/options/src/track_propagation.cpp

```diff
@@ -23,6 +23,8 @@ namespace po = boost::program_options;
 
 track_propagation::track_propagation()
     : interface("Track Propagation Options") {
+    m_search_window[0] = config.navigation.search_window[0];
+    m_search_window[1] = config.navigation.search_window[1];
 
     m_desc.add_options()("constraint-step-size-mm",
                          po::value(&(config.stepping.step_constraint))
```

Comment:
Can you please also add the config default values as the io default values for the rest of this file?

Reply:
Updated to draw all the configuration parameters from the detray default.